jira/src/main/kotlin/gropius/sync/jira/model/IssueData.kt

```diff
@@ -52,10 +53,27 @@ class JiraTimelineItem(val id: String, val created: String, val author: JsonObje
         return id
     }
 
+    fun isStringAndNullTemplate(encodedTypes: String): Pair<Boolean, Boolean> {
+        val typedef = Json.parseToJsonElement(encodedTypes).jsonObject
+        return when {
+            typedef.containsKey("type") -> (typedef["type"]!!.jsonPrimitive.content == "string") to (typedef.containsKey(
+                "nullable"
+            ) && typedef["nullable"]!!.jsonPrimitive.content == "true")
+
+            typedef.containsKey("enum") -> false to false
+            typedef.containsKey("elements") -> false to false
+            typedef.containsKey("properties") -> false to false
+            typedef.containsKey("values") -> false to false
+            typedef.containsKey("discriminator") -> false to false
+            else -> true to true
```

Comment:
Why is the else case true,true?

Reply:
empty object means any. you wanted json typedef, you get json typedef

Reply:
we do not support that type
besides, I would prefer if you use the Schema class and then use the `nullable` and `form` properties
you can use `ObjectMapper.readTree` to get the object
see TemplatedNodeService

Reply:
fa99ad45520a153b257aca5f0d0ba0914308f9e6